pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -1022,7 +1022,7 @@ func TestAssignFlavors(t *testing.T) {
 					Flavors: []cache.FlavorQuotas{{
 						Name: "one",
 						Resources: map[corev1.ResourceName]*cache.ResourceQuota{
-							corev1.ResourceCPU: {Nominal: 2000, BorrowingLimit: pointer.Int64(8_000)},
+							corev1.ResourceCPU: {Nominal: 2000, BorrowingLimit: ptr.To(int64(8_000))},
```

Comment:
```suggestion
							corev1.ResourceCPU: {Nominal: 2000, BorrowingLimit: ptr.To[int64](8_000)},
```

Reply:
thank you.  I knew I missed some.  Applied.